applications/openshift/registry/ocp_allowed_registries/rule.yml

```diff
@@ -8,6 +8,15 @@ description: |-
     and pods. This configuration setting ensures that all registries other than
     those specified are blocked.
 
+    You can set the allowed repositories by applying the following manifest using
+    <code>oc patch</code>:
```

Comment:
We might want to include the patch command itself just for ease-of-use

Reply:
Done, does it look any better now?